pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -361,7 +356,35 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 			ObjectMeta: kapi.ObjectMeta{
 				Name: JobControllerRoleName,
 			},
-			Rules: []authorizationapi.PolicyRule{},
+			Rules: []authorizationapi.PolicyRule{
+				// JobController.jobController.ListWatch
+				{
+					APIGroups: []string{authorizationapi.APIGroupExtensions},
+					Verbs:     sets.NewString("list", "watch"),
+					Resources: sets.NewString("jobs"),
+				},
+				// JobController.syncJob() -> updateJobStatus()
+				{
+					APIGroups: []string{authorizationapi.APIGroupExtensions},
+					Verbs:     sets.NewString("update"),
+					Resources: sets.NewString("jobs/status"),
+				},
+				// JobController.podController.ListWatch
+				{
+					Verbs:     sets.NewString("list", "watch"),
+					Resources: sets.NewString("pods"),
+				},
+				// JobController.podControl (RealPodControl)
+				{
+					Verbs:     sets.NewString("create", "delete"),
+					Resources: sets.NewString("pods"),
+				},
+				// JobController.podControl.recorder
+				{
+					Verbs:     sets.NewString("create", "update"),
```

Comment:
Why patch is needed? If that's true patch will be also needed in rc controller,  since both controllers create event recorder identically.